nima/tests/integration/http2/server/src/test/java/io/helidon/nima/tests/integration/http2/webserver/Http2ErrorHandlingWithOutputStreamTest.java

```diff
@@ -172,6 +174,7 @@ void testGetOutputStreamWriteTwiceThenError_expect_invalidResponse() {
     }
 
     @Test
+    @DisabledOnOs(OS.WINDOWS)
```

Comment:
why?  Can't we fix the core issue in this PR?  If not we need to add more comments w/ a tracking issue.